db/models.py

```diff
@@ -1 +1,26 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name = models.CharField(primary_key=True, max_length=255)
+    description = models.TextField(blank=True)
+
+
+class Skill(models.Model):
+    name = models.CharField(primary_key=True, max_length=255)
+    bonus = models.CharField(max_length=255)
+    race = models.ForeignKey(Race, on_delete=models.PROTECT)
+
+
+class Guild(models.Model):
+    name = models.CharField(primary_key=True, max_length=255)
+    description = models.TextField(blank=True, null=True)
+
+
+class Player(models.Model):
+    nickname = models.CharField(primary_key=True, max_length=255)
+    email = models.EmailField(max_length=255)
+    bio = models.CharField(max_length=255)
+    race = models.ForeignKey(Race, on_delete=models.PROTECT)
+    guild = models.ForeignKey(Guild, on_delete=models.SET_NULL, null=True)
```

Comment:
add `related_name`, it's a best practice 😸

Reply:
Please do not add this. Yes, this is a best practice, but the tests are not ready for such code(
Sorry...